tasks/__init__.py

```diff
@@ -79,7 +84,11 @@ def git_logs():
 def apiserver(port=8000, wait=True, host='127.0.0.1'):
     """Run the API server."""
     env = os.environ.copy()
-    cmd = 'DJANGO_SETTINGS_MODULE=api.base.settings {} manage.py runserver {}:{} --nothreading'.format(sys.executable, host, port)
+    cmd = 'DJANGO_SETTINGS_MODULE=api.base.settings {} manage.py runserver {} --nothreading'.format(sys.executable, port)
```

Comment:
Discussed in person: `host` was not there then, looks like merge conflicts, will fix this.